x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -94,67 +84,161 @@ function newLayerState(layerId: string): LayerConfig {
     seriesType: 'bar_stacked',
     accessors: [generateId()],
     title: '',
-    showGridlines: false,
-    position: Position.Left,
     splitAccessor: generateId(),
   };
 }
 
+interface LocalState {
+  isChartOptionsOpen: boolean;
+  openLayerId: string | null;
+}
+
 export function XYConfigPanel(props: VisualizationProps<State>) {
   const { state, setState, frame } = props;
 
+  const [localState, setLocalState] = useState({
+    isChartOptionsOpen: false,
+    openLayerId: null,
+  } as LocalState);
+
   return (
     <EuiForm className="lnsConfigPanel">
+      <EuiFormRow>
+        <EuiPopover
+          id="lnsXY_chartConfig"
+          isOpen={localState.isChartOptionsOpen}
+          closePopover={() => {
+            setLocalState({ ...localState, isChartOptionsOpen: false });
+          }}
+          button={
+            <EuiFormRow>
+              <>
+                <EuiButton
+                  iconType="gear"
+                  size="s"
+                  data-test-subj="lnsXY_chart_settings"
+                  onClick={() => {
+                    setLocalState({ ...localState, isChartOptionsOpen: true });
+                  }}
+                >
+                  <FormattedMessage
+                    id="xpack.lens.xyChart.chartSettings"
+                    defaultMessage="Chart Settings"
+                  />
+                </EuiButton>
+              </>
+            </EuiFormRow>
+          }
+        >
+          <>
+            <EuiFormRow>
+              <EuiSwitch
+                label={i18n.translate('xpack.lens.xyChart.isHorizontalSwitch', {
```

Comment:
I think this is really nice - probably the best way to show this concept to the user